src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java

```diff
@@ -536,7 +538,7 @@
 
 	StringEditorCell createStringEditorCell();
 	
-	JSONListCellEditor createListCellEditor();
+	JSONListCellEditor createJSONListCellEditor();

```

Comment:
Renamed injector method to be more clear (since we have a ListCellEditor as well)